swan-lake/learn-the-platform/source-code-dependencies/manage-dependencies.md

```diff
@@ -153,6 +157,57 @@ Ballerina considers the version specified in the Ballerina.toml as the minimum r
 However, the compiler gives priority to the latest version if a new patch version is found in distribution or Ballerina Central repositories.
 At this point, the compiler resolves the latest version and ignores the dependency version in the local repository.
 
+
+## Use Maven repositories for the dependency management
+
+Currently, [Nexus](https://www.sonatype.com/products/sonatype-nexus-repository), [Artifactory](https://jfrog.com/artifactory/) and [Github packages](https://github.com/features/packages) are only supported. To use this feature `Settings.toml` needs to be configured with the proper repository configurations.
```

Comment:
Ballerina supports [Nexus](https://www.sonatype.com/products/sonatype-nexus-repository), [Artifactory](https://jfrog.com/artifactory/) and [Github packages](https://github.com/features/packages) to be set up as custom repositories. You can configure the custom repository in `<USER_HOME>/.ballerina/Settings.toml` to integrate it to package resolution.